### ISSUE TITLE
Add an option to copy dictionary path by clicking the extension in JSON

### ISSUE THREAD

**Issue by mathewvarghesemanu** [richie5um/vscode-statusbar-json-path#26]
In python, a.b.c doesnot work for accessing data at c in json string or dictionary, which is the most preferred format for dealing with json data in python. I request to give an option to choose a["b"]["c"] always, not just when there are spaces between the words in a key.

**Comment by QuentiumYT** [comment]
Hello, I added this feature on my fork, https://github.com/Quentium-Forks/vscode-statusbar-json-path
I didn't published on VSCode but I built it in the release section.

Hope it help :)

**Comment by richie5um** [comment]
Nice. I’m happy to take a PR if you want that in the main version. ?!

**Comment by QuentiumYT** [comment]
Good evening, I'd appreciate a lot that others will have this option. I didn't create a PR since I linted all of your code and it sometimes annoys the author. I also changed a few names and stuff but I can clean that up if needed :)

**Comment by mathewvarghesemanu** [comment]
I appreciate @QuentiumYT and @richie5um for the quick reply. 

This would be an amazing feature and would help a lot of people, given python is gaining popularity at a fast pace, especially in the AI/ML space.

**Comment by richie5um** [comment]
@QuentiumYT "since I linted all of your code" <= tabs / spaces ? ;-)

I rarely have time to do any changes to my open-source stuff, so I'm happy to take a PR if you have one.

**Comment by QuentiumYT** [comment]
Yes, I changed to 2 spaces for JS convention :p I will create a new clean branch for the merge soon, I do have few things planned these next days
